main.css

```diff
@@ -0,0 +1,70 @@
+* {
+    border: 1px solid gold;
+ }
+
+p {
+    font: sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font: serif;
+    text-align: center;
+}
+
+a {
+    text-decoration: none;
+    color: #4242EA;
+}
+
+ul {
+    list-style: none;
+}
+
+footer span {
+    font-weight: 700;
+}
+
+.weather p {
+    font-size: 40px;
+}
+
+.above-the-fold article {
+    display: inline-block;
+    justify-content: flex-end;
```

Comment:
Note that justify-content won't do anything since this element isn't a flex element (made to be an inline-block element above)